kong/tools/gzip.lua

```diff
@@ -15,7 +16,16 @@ local GZIP_CHUNK_SIZE = 65535
 
 
 local function read_input_buffer(input_buffer)
+  local count = 0
+  local yield_size = GZIP_CHUNK_SIZE * 2
```

Comment:
Is there any consideration to yield for `2 GZIP_CHUNK_SIZE`?

Reply:
We don't have enough data to support this number, so `GZIP_CHUNK_SIZE * 2` is just a estimated value.